src/util/errors-ts.ts

```diff
@@ -1072,3 +1072,30 @@ export class MissingDotenvVarsError extends NowError<
     });
   }
 }
+
+export class UnauthorizedRequestError extends NowError<
+  'UNAUTHORIZED_REQUEST_ERROR',
+  { detail: string; type: string; cns: string[] }
+> {
+  constructor(detail: string, type: string, cns: string[]) {
+    super({
+      code: 'UNAUTHORIZED_REQUEST_ERROR',
+      meta: { detail, type, cns },
+      message: 'ACME request was unauthorized'
+    });
+  }
+}
+
+export class DNSError extends NowError<
+  'DNS_ERROR',
+  { detail: string; type: string; cns: string[] }
+> {
+  constructor(detail: string, type: string, cns: string[]) {
+    super({
+      code: 'DNS_ERROR',
+      meta: { detail, type, cns },
+      message:
+        'There was a problem with a DNS query during identifier validation'
```

Comment:
I think we can provide more context here to help guide the user in resolving the DNS issue